XamarinCommunityToolkitSample/Pages/Behaviors/EventToCommandBehaviorPage.xaml

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:vm="clr-namespace:XamarinCommunityToolkitSample.ViewModels.Behaviors"
+             xmlns:behaviors="clr-namespace:XamarinCommunityToolkit.Behaviors;assembly=XamarinCommunityToolkit"
+             x:Class="XamarinCommunityToolkitSample.Pages.Behaviors.EventToCommandBehaviorPage">
+
+    <ContentPage.BindingContext>
+        <vm:EventToCommandBehaviorViewModel />
+    </ContentPage.BindingContext>
+
+    <ContentView Padding="{StaticResource ContentPadding}">
+        <StackLayout VerticalOptions="CenterAndExpand"
```

Comment:
I'm missing the descriptive label on this page. What is the user looking at here?

Reply:
added